inc/endpoints/class-token.php

```diff
@@ -32,7 +33,7 @@ public function register_routes() {
 					'validate_callback' => 'rest_validate_request_arg',
 				],
 			],
-		] );
+		]);
```

Comment:
The changes in this file aren't correct, and in any case don't appear to be related to this PR.